src/static/js/modules/form-explainer.js

```diff
@@ -451,15 +455,45 @@ $(document).ready(function(){
     }
     if ( typeof $target !== 'undefined' ) {
       // remove class from all
-      $('.expandable__form-explainer, .image-map_overlay').removeClass('has-attention');
-      if ( $target.hasClass('has-attention') ) {
-        $target.removeClass('has-attention');
+      $('.expandable__form-explainer, .image-map_overlay').removeClass('hover-has-attention');
+
+      if ( $target.hasClass('hover-has-attention') ) {
+        $target.removeClass('hover-has-attention');
+      
+        if ( $target.hasClass('has-attention')) {
+          $target.removeClass('has-attention');
+        }
       } else {
-        $target.addClass('has-attention');
+        $target.addClass('hover-has-attention');
+      }
+    }
+  });
+
+  // When mousing over a term or highlighted area of the image map,
+  // call attention to the associated map area or term, respectively.
+  $WRAPPER.on( 'keypress click', '.image-map_overlay, .expandable__form-explainer', function( event ) {
```

Comment:
It looks like this click event isn't getting fired on the expandables, probably because the event handler in the expandable code stops propagation when the `.expandable_target` is clicked. 

I just added a click event directly on the `.expandable_target` class to handle post-animation image positioning, and there's also another click handler on `.image-map_overlay` that toggles the expandables. Since we need to have separate event handlers on these two classes for other behaviors, I'm wondering if we could move this code out into a function and call it from those other handlers.